src/tree/activityLog/ActivityItem.ts

```diff
@@ -71,10 +82,10 @@ export class ActivityItem implements TreeElementBase, Disposable {
     public error?: unknown;
     private latestProgress?: { message?: string };
 
-    public constructor(activity: Activity) {
+    public constructor(readonly activity: Activity) {
         this.id = activity.id;
         this.setupListeners(activity);
-        this.startedAtMs = Date.now();
+        this.startedAtMs = this.activity.startTime?.getTime() ?? Date.now();
```

Comment:
I don't actually see this being used anywhere. We can probably just remove this property and use the `activity.startTime`.

It also doesn't make sense that it used to set `this.startedAtMs` on construction _and_ `onStart`, overwriting the time.

Reply:
Yeah I think this was a legacy implementation from when we used `compareChildrenImpl` for v1.